app/components/ui-table/cell/events/view/tickets/access-codes/cell-actions.js

```diff
@@ -0,0 +1,4 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+});
```

Comment:
@pradeepgangwar this file is not needed. It can be removed I think.

Reply:
@ritikamotwani removed the file.